assets/src/scss/new-identity/style.scss

```diff
@@ -42,4 +42,8 @@
   --button-secondary--hover--background: var(--p4-dark-green-800);
   --button-secondary--hover--border-color: var(--p4-dark-green-800);
   --button-secondary--hover--color: var(--white);
+  --link--color: var(--grey-900);
+  --link--visited--color: var(--grey-900);
+  --link--hover--color: var(--grey-900);
+  --link--hover--text-decoration: none;
```

Comment:
Are you talking about the links at the top of the post, like "Press Release"? These are contextual links, they are [supposed to be green](https://jira.greenpeace.org/browse/PLANET-6991) in the new identity 🙂

Reply:
Oh, my bad I see that. But it still does not affect regular links.

Reply:
I do see the changes apply to regular links 🤔 Something seems to be wrong on your local @Osong-Michael!
<img width="815" alt="Screenshot 2023-03-30 at 08 45 01" src="https://user-images.githubusercontent.com/6949075/228751875-af226fd3-28cb-46ba-8e0d-4aba89086251.png">
@dantovbein I would add the bottom border and background on hover, I can't think of a case where the design team wouldn't want them since the way it is now you cannot tell that they are links

Reply:
😭